src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchive.cs

```diff
@@ -30,92 +35,60 @@ public class ZipArchive : IDisposable
         private byte[]? _archiveComment;
         private Encoding? _entryNameEncoding;
 
-#if DEBUG_FORCE_ZIP64
-        public bool _forceZip64;
-#endif
```

Comment:
This doesn't look like something that should be removed in this PR.

Reply:
Good catch. I've been working on https://github.com/carlossanlop/DocsPortingTool/issues/65, but we'll have to manually fix it here.